### ISSUE TITLE
SceneKit SDK Feature Requests

### ISSUE THREAD

**Issue by avi-c** [mapbox/mapbox-scenekit#69]
Hello everyone! The team is laying out the roadmap for the SDK, looking forward to an official v1.0 release. We’ve drafted a list of items that should be done.

In the interests of transparency and meeting the needs of you, the developer, I am adding a list of some of our work items below and would welcome your input on the relative priority of items from your perspective. Feel free to comment and also add any items you see as important.

A partial list of items are:

  - Scaffolding for proper v1.0 SDKs (CI, develop branch, etc.)
  - Better support for custom Mapbox Studio map style URLs
  - Crash fixes for some occasional crashes that we’ve seen
  - Revise API. Switch to delegate pattern for texture/heightmap loading instead of callbacks. We are open to feedback from our users on the best pattern for you, but this is our plan. 
  - Increased integration with Mapbox Mobile iOS SDK
  - Additional sample projects

Once we have a settled list, we’ll be cutting tickets for each item. Please provide feedback below. We want to make sure we prioritize the changes that would help you the most.

**Comment by fanquake** [comment]
Hi @avi-c, are there any updates to the roadmap/list or somewhere else to follow for updates?

**Comment by hyerra** [comment]
Yeah, I have been kind of wondering that too. It seems like this project has mainly been stale for the most part.
